Origin: https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=073661773872709518d35d4d093f3a715281f21d
Bug-Debian: https://bugs.debian.org/961278
Description: ip route: print route type in JSON output
 ip route generates an invalid JSON if the route type has to be printed,
 eg. when detailed mode is active, or the type is different that unicast:
--- a/ip/iproute.c
+++ b/ip/iproute.c
@@ -766,7 +766,7 @@
 
 	if ((r->rtm_type != RTN_UNICAST || show_details > 0) &&
 	    (!filter.typemask || (filter.typemask & (1 << r->rtm_type))))
-		print_string(PRINT_ANY, NULL, "%s ",
+		print_string(PRINT_ANY, "type", "%s ",
 			     rtnl_rtntype_n2a(r->rtm_type, b1, sizeof(b1)));
 
 	color = COLOR_NONE;
